Assets/Plugins/Source/Editor/ConfigEditors/PlatformConfigEditor.cs

```diff
@@ -54,12 +58,26 @@ public virtual void RenderOverrides()
             GUILayout.Label($"{PlatformManager.GetFullName(Platform)} Override Configuration Values",
                 EditorStyles.boldLabel);
 
+            GUIEditorUtility.AssigningFlagTextField("Integrated Platform Management Flags (Separated by '|')", ref config.overrideValues.integratedPlatformManagementFlags, 345);
+
             GUIEditorUtility.AssigningFlagTextField("Override Platform Flags (Separated by '|')", ref config.overrideValues.platformOptionsFlags, 250);
 
             GUIEditorUtility.AssigningFloatToStringField("Override initial button delay for overlay", ref config.overrideValues.initialButtonDelayForOverlay, 250);
 
             GUIEditorUtility.AssigningFloatToStringField("Override repeat button delay for overlay", ref config.overrideValues.repeatButtonDelayForOverlay, 250);
 
+            // TODO: As far as can be determined, it appears that the following
+            //       values are the only ones within "overrideValues" that are
+            //       actually being used to override the otherwise defined
+            //       values within EOSConfig. Changing the values in the fields
+            //       for which the input fields are rendered above does not 
+            //       seem to have any affect. 
+            // 
+            //       This is a bug, but is not relevant to the current task as 
+            //       of this writing, which is to simply add the field member
+            //       "integratedPlatformManagementFlags" to EOSConfig, and make
+            //       sure that field is editable within the editor.
```

Comment:
Part of my thought process for not fixing this here is because I started to fix it - and found that doing so is actually not straightforward, and the work would need to be redone in light of the changes we are planning to make to Config validation and platform variation.

Basically, I determined that the time it would take to fix it the way it is was greater than the time it would take to continue on the current development plan.